### PR TITLE
Ecm 140 fixes

### PR DESCRIPTION
**Pull request by gauravchawla73463**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java [hunk 1]
```diff
@@ -69,6 +71,60 @@ private void executeReport(ListingDetails listingDetails, List<SubmitEvent> subm
...
+        adhocReportType.setSubSplit(ZERO);
+        adhocReportType.setCaseReference(ZERO);
+        adhocReportType.setLeadCase(ZERO);
+        adhocReportType.setHearingType(ZERO);
+        adhocReportType.setJudicialMediation(ZERO);
+        adhocReportType.setHearingTelConf(ZERO);
+        adhocReportType.setHearingDuration(ZERO);
+        adhocReportType.setHearingClerk(ZERO);
+
+        String hearingNumber = ZERO + "|"
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java:96]: This should be something else...a class constant

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java [hunk 2]
```diff
@@ -69,6 +71,60 @@ private void executeReport(ListingDetails listingDetails, List<SubmitEvent> subm
 
     }
 
+    private void initReport(ListingDetails listingDetails) {
+
+        var adhocReportType = new AdhocReportType();
+
+        //LocalReportsSummary fields
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java:78]: Why put a comment when your code can document itself? Create a method called initSummaryFields. It'll make initReport easier to understand too

Comment (gauravchawla73463) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java:78]: refactored

**Comment by Angela-LW** [comment]
Fyi I'm not gonna make you rewrite the whole thing, but in the future I would prefer if we moved away from using the the adhocreport object to more specific and meaningful object models.

**Review by Angela-LW — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java (hunk 1 above)
Comment (gauravchawla73463) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java]: done

**Review by jthmcts — APPROVED** [review]

**Pull request merged**